src/main/webapp/js/actions/auth-actions.js

```diff
@@ -25,6 +29,46 @@ export const authError = error => ({
   payload: error,
 });
 
+export const signInOffline = ({ password, savedLogin }, callback) => (dispatch) => {
```

Comment:
all of this should be handled by the redux offline, you will need to update the redux offline configuration for this